hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/common/fica/FiCaSchedulerApp.java

```diff
@@ -615,7 +615,7 @@ public boolean apply(Resource cluster, ResourceCommitRequest<FiCaSchedulerApp,
                 allocation.getAllocationLocalityType(),
                 schedulerContainer.getSchedulerNode(),
                 schedulerContainer.getSchedulerRequestKey(),
-                schedulerContainer.getRmContainer().getContainer());
+                  schedulerContainer.getRmContainer());
```

Comment:
Can you confirm this 
schedulerContainer.getRmContainer().getContainer()
**returns** 
  Container getContainer();
------------------------------------------------------
  public RMContainer getRmContainer() {
    return rmContainer;
  }